grobid-core/src/main/java/org/grobid/core/sax/PDFALTOOutlineSaxHandler.java

```diff
@@ -53,11 +54,14 @@ public void endElement(java.lang.String uri, java.lang.String localName,
 			java.lang.String qName) throws SAXException {
 
 		if (qName.equals("STRING")) {
-			label = getText();
+		    currentNode.setLabel(getText());
+            currentNode.setBoundingBox(box);
 		} else if (qName.equals("ITEM")) {
-			currentNode.setLabel(label);
-			currentNode.setBoundingBox(box);
-		}
+            box = null;
+            label = null;
+		} else if (qName.equals("TOCITEMLIST")) {
+		    currentParentId = -1;
```

Comment:
This is actually  50% of the origin of the error...  currentParentId was not reinit.